extensions/2.0/Khronos/KHR_materials_sheen/schema/glTF.KHR_materials_sheen.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "KHR_materials_sheen glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the sheen material model.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "sheenColorFactor": {
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0.0,
+                "maximum": 1.0
+            },
+            "minItems": 3,
+            "maxItems": 3,
+            "description": "Color of the sheen layer (in linear space).",
+            "default": [0, 0, 0]
+        },
+        "sheenTexture": {
```

Comment:
Would it be OK to rename `sheenTexture` to `sheenColorTexture`?  That way we keep the factor+texture naming convention going.

Reply:
Nevermind, roughness is now bundled here.